pkg/api/controller_test.go

```diff
@@ -2437,6 +2434,38 @@ func TestController_LinkPhysicalAddressHandler(t *testing.T) {
 			t.Fatalf("LinkPhysicalAddress status code: %d, expected: %d", resp.HTTPResponse.StatusCode, expectedStatusCode)
 		}
 	})
+
```

Comment:
🥇

Reply:
Maybe I missed it but can we add a test to try and link an expired physical address?

Reply:
I can't think of a trivial way to generate an expired address, so I think it would be out of scope for this PR.

Reply:
@N-o-Z - added the test